### PR TITLE
Fix meilisearch test flakyness

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/integration/test_storage_meilisearch/test.py [hunk 1]
```diff
@@ -374,16 +383,18 @@ def test_insert_secure(started_cluster):
...
+    node.query("DROP TABLE IF EXISTS new_table")
     node.query(
         "CREATE TABLE new_table(id UInt64, data String) ENGINE = MeiliSearch('http://meili_secure:7700', 'new_table', 'password')"
     )
 
     node.query(
         "INSERT INTO new_table (id, data) VALUES (1, '1') (2, '2') (3, '3') (4, '4') (5, '5') (6, '6') (7, '7')"
     )
-    sleep(1)
+    sleep(5)
```
Comment (SmitaRKulkarni) [inline tests/integration/test_storage_meilisearch/test.py:394]: why are we increasing the sleep time ?

Comment (kssenii) [reply tests/integration/test_storage_meilisearch/test.py:394]: because 1 is not enough

Comment (SmitaRKulkarni) [reply tests/integration/test_storage_meilisearch/test.py:394]: are we waiting for some operation to complete ? so executing INSERT statement is not enough for next checks of meili client? Trying to understand how having more sleep time would help

Comment (kssenii) [reply tests/integration/test_storage_meilisearch/test.py:394]: it is specific to Meilisearch, it does several operations asynchronously, including updating documents in the index, which is checked in this test.

**Comment by kssenii** [comment]
> Stateful tests (debug) — fail: 1

https://github.com/ClickHouse/ClickHouse/pull/46593

> Integration tests (asan) [4/6] — fail: 1

https://github.com/ClickHouse/ClickHouse/pull/46595

> Integration tests (release) [2/4] — fail: 1

test_merge_tree_s3/test.py::test_attach_detach_partition[node]

File: tests/integration/test_storage_meilisearch/test.py (hunk 1 above)
Comment (kssenii) [reply tests/integration/test_storage_meilisearch/test.py:394]: https://docs.meilisearch.com/learn/advanced/asynchronous_operations.html#which-operations-are-asynchronous

**Review by SmitaRKulkarni — APPROVED** [review]
LGTM

**Pull request merged**